apps/web/package.json

```diff
@@ -46,8 +48,11 @@
     "autoprefixer": "^10.4.19",
     "drizzle-kit": "^0.22.8",
     "eslint": "^8.57.0",
+    "node-mocks-http": "^1.15.0",
     "postcss": "^8.4.39",
     "tailwindcss": "^3.4.4",
-    "typescript": "^5.5.3"
+    "typescript": "^5.5.3",
+    "vite-tsconfig-paths": "^4.3.2",
```

Comment:
Vitest has native alias config. Can you try this?

<img width="658" alt="image" src="https://github.com/user-attachments/assets/c3ae8dcc-7f5a-426a-b4ac-cc28d0ce6919">

Reply:
hmm unable to fix this with the plugin nor with your suggestion. Any more ideas?

```
Error: Failed to load url $core/services/users/createUser (resolved id: $core/services/users/createUser) in /Users/gerardclos/Documents/work/latitude-llm/packages/core/src/tests/factories/users.ts. Does the file exist?
```

Otherwise I will have to copy/paste factories across projects which is far from ideal tbh

Reply:
Carlos solved with the plugin I think

Reply:
Wait, I used your branch
<img width="468" alt="image" src="https://github.com/user-attachments/assets/22e24a4e-1bf9-4f3c-8bf2-64cef756f0e3">


<img width="515" alt="image" src="https://github.com/user-attachments/assets/f14c90c4-fc8e-4aae-a373-1c1700368c04">


<img width="477" alt="image" src="https://github.com/user-attachments/assets/5e9faf48-3d00-477b-a410-14a5bb89e321">

Reply:
:/ will try it later

Reply:
Didn't work that way?

Reply:
it works 👌🏼 I was using relative paths this is why it wasn't working

Reply:
@csansoon change yours ;)